parsl/providers/condor/condor.py

```diff
@@ -230,8 +231,10 @@ def submit(self, command, blocksize, tasks_per_node, job_name="parsl.auto"):
         channel_script_path = self.channel.push_file(script_path, self.channel.script_dir)
 
         cmd = "condor_submit {0}".format(channel_script_path)
-        retcode, stdout, stderr = super().execute_wait(cmd)
-        logger.debug("Retcode:%s STDOUT:%s STDERR:%s", retcode, stdout.strip(), stderr.strip())
+        try:
+            retcode, stdout, stderr = super().execute_wait(cmd)
+        except Exception as e:
+            raise ScaleOutFailed(self.label, str(e))
```

Comment:
losing the underlying exception info (such as stacktrace) for `e` gives me a slightly bad feeling. Maybe that's a criticism of ScaleOutFailed though.